From: https://github.com/MariaDB/server/commit/0fc0eb1e5b9015e51daef9709937be69e9e7c4f7
Author: Daniel Black <daniel@mariadb.org>
Date: Mon, 16 Nov 2020 08:48:32 +1100
Subject: [PATCH] MDEV-24124: main.drop test - mulitarch/os error messages
 Account for variety of mips, hppa, solaris and other messages.
 .
 Copied from rpl.rpl_drop_db test.
 .
 This can be removed once 10.5.9 is imported.
---
 mysql-test/main/drop.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mysql-test/main/drop.test
+++ b/mysql-test/main/drop.test
@@ -187,7 +187,7 @@ write_file $MYSQLD_DATADIR/mysql_test/do
 do_not_delete
 EOF
 
-replace_result $MYSQLD_DATADIR ./ \\ / 41 39;
+replace_result $MYSQLD_DATADIR ./ \\ / 66 39 93 39 17 39 247 39 41 39 "File exists" "Directory not empty";
 error ER_DB_DROP_RMDIR;
 drop database mysql_test;
 select database();
